src/themeMode/ThemeMode.tsx

```diff
@@ -0,0 +1,13 @@
+import { THEME_MODE_KEY } from '../constants/storage';
```

Comment:
Move this file to config directory
As it is **not** a react component, name it by lowercase